package.json

```diff
@@ -181,7 +181,7 @@
     "@stacks/profile": "6.15.0",
     "@stacks/rpc-client": "1.0.3",
     "@stacks/storage": "6.15.0",
-    "@stacks/transactions": "6.17.0",
+    "@stacks/transactions": "7.0.2",
```

Comment:
I am hitting a lot of errors with `@stacks/transactions` imports with this change, what am I missing?

Reply:
All good if I revert it back...

![Screenshot 2024-11-30 at 1 23 40 PM](https://github.com/user-attachments/assets/7e53c2ac-b07c-4a22-989a-72e9dddd775a)

Reply:
I see we prob need lib update for the sbtc tx. Still looking thru this so ignore me, haha.